src/pint/fitter.py

```diff
@@ -160,13 +160,27 @@ def get_summary(self, nodmx=False):
         import uncertainties.umath as um
         from uncertainties import ufloat
 
+        # Check if Wideband or not
+        if self.resids.__class__.__name__ == "WidebandTOAResiduals":
+            is_wideband = True
+        else:
+            is_wideband = False
+
         # First, print fit quality metrics
         s = "Fitted model using {} method with {} free parameters to {} TOAs\n".format(
             self.method, len(self.model.free_params), self.toas.ntoas
         )
-        s += "Prefit residuals Wrms = {}, Postfit residuals Wrms = {}\n".format(
-            self.resids_init.rms_weighted(), self.resids.rms_weighted()
-        )
+        if is_wideband:
+            s += "Prefit TOA residuals Wrms = {}, Postfit TOA residuals Wrms = {}\n".format(
```

Comment:
You can use f-strings instead of `.format()`:
```python
s += f"Prefit TOA residuals Wrms = {self.resids_init.toa.rms_weighted()}"
```

Reply:
Done!